src/js/pages/Escrow/components/CardEscrowBuyer.jsx

```diff
@@ -107,13 +107,33 @@ PreFund.propTypes = {
   statusContactCode: PropTypes.string
 };
 
+const Dispute = () => (
+  <React.Fragment>
+    <RoundedIcon icon={faStop} bgColor="red"/>
+    <h2 className="mt-4">Disputed</h2>
+    <p>A dispute has been opened for this trade</p>
+  </React.Fragment>
+);
+
 class CardEscrowBuyer extends Component {
   render(){
-    const {trade, payStatus, payAction, rateTransaction} = this.props;
+    const {trade, payStatus, payAction, rateTransaction, arbitrationDetails} = this.props;
 
     const showLoading = payStatus === States.pending;
     const showWaiting = payStatus === States.success || trade.status === escrow.helpers.tradeStates.released;
 
+    if(arbitrationDetails && (arbitrationDetails.open || arbitrationDetails.result !== "0")){
```

Comment:
Is there a chance that the result could be an integer instead of a string?

Reply:
I'll add a `.toString()`  just in case there's the possibility of it being a number